cmd/osctl/cmd/root.go

```diff
@@ -55,8 +56,14 @@ func Execute() {
 	var globalCtxCancel context.CancelFunc
 
 	globalCtx, globalCtxCancel = context.WithCancel(context.Background())
+
 	defer globalCtxCancel()
 
+	// Update context with grpc metadata for proxy/relay requests
```

Comment:
Will make another PR to clean up the os api bits that explicitly set the metadata in the `cmd/osctl/cmd/x.go` files so we can just make use of this.